packages/e2e/test/wallet/environment.ts

```diff
@@ -3,7 +3,6 @@ import * as envalid from 'envalid';
 export const env = envalid.cleanEnv(process.env, {
   ASSET_PROVIDER: envalid.str(),
   ASSET_PROVIDER_PARAMS: envalid.json({ default: {} }),
-  BLOCKFROST_API_KEY: envalid.str(),
   CHAIN_HISTORY_PROVIDER: envalid.str(),
```

Comment:
The API key presence validation is already done by the factories, so this is not needed here

```javascript
/**
 * Gets the singleton blockfrost API instance.
 *
 * @returns The blockfrost API instance, this function always returns the same instance.
 */
const getBlockfrostApi = async () => {
  if (blockfrostApi !== undefined) return blockfrostApi;

  if (process.env.BLOCKFROST_API_KEY === undefined) throw new Error(BLOCKFROST_MISSING_PROJECT_ID);

  return new BlockFrostAPI({ isTestnet: true, projectId: process.env.BLOCKFROST_API_KEY });
};
```